### PR TITLE
Markup improvements & new example

### PR DESCRIPTION
**Pull request by lingfish**
Some markup changes for readability, and a new example.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -112,74 +122,74 @@ Objects Interface
 
 All top level objects support interface for CRUD operations. List of supported objects is defined in next section.
 
-- create(cls, connector, check_if_exists=True, update_if_exists=False, \**kwargs)
+- ``create(cls, connector, check_if_exists=True, update_if_exists=False, \**kwargs)``
```
Comment (yuewko) [inline README.rst]: The `\` in front of `**kwargs` is now showing up in the rendered version.

File: README.rst [hunk 2]
```diff
@@ -112,74 +122,74 @@ Objects Interface
...
     Creates object on NIOS side.
-    Requires connector passed as the first argument, check_if_exists and update_if_exists are optional.
-    Object related fields are passed in as kwargs: field=value, field2=value2.
+    Requires connector passed as the first argument, ``check_if_exists`` and ``update_if_exists`` are optional.
+    Object related fields are passed in as kwargs: ``field=value``, ``field2=value2``.
     
-- search(cls, connector, return_fields=None, search_extattrs=None, force_proxy=False, \**kwargs)
+- ``search(cls, connector, return_fields=None, search_extattrs=None, force_proxy=False, \**kwargs)``
```
Comment (yuewko) [inline README.rst]: ditto

File: README.rst [hunk 3]
```diff
@@ -112,74 +122,74 @@ Objects Interface
...
+    ``return_fields`` can be set to retrieve particular fields from NIOS,
+    for example ``return_fields=['view', 'name']``.
+    If ``return_fields`` is ``[]`` default ``return_fields`` are returned by NIOS side for current ``wapi_version``.
+    ``search_extattrs`` is used to filter out results by extensible attributes.
+    ``force_proxy`` forces search request to be processed on Grid Master (applies only in cloud environment)
     
-- search_all(cls, connector, return_fields=None, search_extattrs=None, force_proxy=False, \**kwargs)
-    Search all objects on NIOS side that match search cryteria. Returns list of objects.
-    All other options are equal to search().
+- ``search_all(cls, connector, return_fields=None, search_extattrs=None, force_proxy=False, \**kwargs)``
```
Comment (yuewko) [inline README.rst]: ditto

**Comment by lingfish** [comment]
Done!

**Comment by lingfish** [comment]
What do you think about the additional example?  Seems like a rather convoluted way to do it.

**Comment by yuewko** [comment]
Regarding the additional example, the same functionality is provided by the `InfobloxObjectManager.create_host_record_from_range()` method.

Maybe start a new section on examples using InfobloxObjectManager and use that as an example?

**Comment by lingfish** [comment]
Are you sure?  Does the range variant accept CIDRs?  I don't think so?

I actually tried using range, and was getting this error:

```
InfobloxCannotCreateObject: Cannot create 'record:host' object(s): { "Error": "AdmConProtoError: 10.0.0.1-10.0.0.254,default does not match any unique networks/ranges (ipv4addr)",
  "code": "Client.Ibap.Proto",
  "text": "10.0.0.1-10.0.0.254,default does not match any unique networks/ranges (ipv4addr)"
} [code 400]
```

**Comment by yuewko** [comment]
You're right. It only accepts `start_ip` and `end_ip`. So, not exactly like your example.

Obviously, you are welcome to add a `create_host_record_from_cidr()` method. ;-)

**Comment by lingfish** [comment]
Ok, are we there yet? ;)

I left my first example in, until you or I add the `create_host_record_from_cidr()` method.  I've kinda spent too much time on these commits, and not enough on my actual code ;)

**Comment by rhossili** [comment]
Perfect!! Ready to merge as far as I'm concerned.

@johnbelamaric , please merge when you have a chance.

**Comment by yuewko** [comment]
@lingfish , @johnbelamaric ... sorry, I was logged in to the wrong account for my last comment.

@johnbelamaric - please merge when you have a chance.

**Pull request merged**